resources/views/tasks.blade.php

```diff
@@ -58,7 +57,7 @@
             <div class="form-group">
                 <label for="time">Time</label>
                 <select class="form-control {{ $errors->first('time') ? 'is-invalid' : '' }}" name="time">
-                    <option selected>Please Select Time</option>>
+                    <option value="" selected>Please Select Time</option>>
```

Comment:
Remove selected from this option

Reply:
I have removed selected form this option